pkg/userdata/flatcar/provider.go

```diff
@@ -374,6 +376,13 @@ storage:
         inline: |
 {{ .NodeIPScript | indent 10 }}
 
+    - path: "/etc/systemd/network/zz-default.network.d/ipv6-fix.conf"
+      filesystem: root
+      mode: 0755
+      contents:
+        inline: |
+{{ .IPv6RouterRA | indent 10 }}
```

Comment:
Do we really need a method/template for this? Seems like there is nothing dynamic and we can just simply hardcode those values here.

Reply:
Good idea.